tests/test_encoder_forward.py

```diff
@@ -78,10 +78,10 @@ def test_forward(self, devices):
             h_out = star_encoder._get_var_params_all(image_ptiles)
 
             # get index matrices
-            locs_mean_indx_mat = star_encoder.indx_mats["loc_mean"]
-            locs_var_indx_mat = star_encoder.indx_mats["loc_logvar"]
-            log_flux_mean_indx_mat = star_encoder.indx_mats["log_flux_mean"]
-            log_flux_var_indx_mat = star_encoder.indx_mats["log_flux_logvar"]
+            locs_mean_indx_mat = star_encoder.loc_mean
+            locs_var_indx_mat = star_encoder.loc_logvar
+            log_flux_mean_indx_mat = star_encoder.log_flux_mean
+            log_flux_var_indx_mat = star_encoder.log_flux_logvar
```

Comment:
yeah it would be best if the attribute was e.g. `loc_mean_indx` to avoid confusion that this is the matrix of indices.